pkg/disk/filesystem.go

```diff
@@ -23,7 +68,7 @@ type Filesystem struct {
 	// The sixth field of fstab(5); fs_passno
 	FSTabPassNo uint64 `json:"fstab_passno,omitempty" yaml:"fstab_passno,omitempty"`
 	// Custom mkfs options
-	MkfsOptions []string `json:"mkfs_options,omitempty" yaml:"mkfs_options,omitempty"`
+	MkfsOptions []MkfsOption `json:"mkfs_options,omitempty" yaml:"mkfs_options,omitempty"`
```

Comment:
I suspect we might end up reverting this change when we decide to support arbitrary mkfs options, but for now I think let's keep it and see how it goes.